src/search/csearch.c

```
@@ -260,58 +302,27 @@ int main(int argc, char *argv[])
+  log_stream = fopen(log_file, "a");
+  if (log_stream == NULL) { OPEN_ERROR(log_file); }
+  fprintf(log_stream, "=> Optimizer: %s\n", optimizer ? optimizer->name : "simplex");
+  sr_optimizer_log_config(log_stream, &opt_cfg);
+  fclose(log_stream);
+
 /***********************************************************************
   Perform the search according to the selected algorithm.
 ***********************************************************************/
 
-  switch(search_type)
+  if (!optimizer) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)